### ISSUE TITLE
Online docs format command argument hyphens (U+002D) as minus sign (U+2212)

### ISSUE THREAD

**Issue by erikarvstedt** [TigerVNC/tigervnc#850]
An example page where this happens: https://tigervnc.org/doc/vncviewer.html
This can be doubly frustrating when copying command line arguments from the docs, because the invalid args are silently ignored by the binaries.

**Comment by samhed** [comment]
Hm, true. At first I thought it might be a browser bug, but when looking at the html we can see that they are in fact specified as `&minus;`:

https://github.com/TigerVNC/tigervnc.github.io/blob/master/doc/vncviewer.html

Judging from the top of that file, these are generated by "groff", so I guess they have a bug..

**Comment by CendioOssman** [comment]
Yeah, not sure we can do much about that. Groff likely only cares about visual output. Have you checked its documentation?

**Comment by samhed** [comment]
Yeah, couldn't find anything about that..

https://www.gnu.org/software/groff/manual/html_node/Groff-Options.html

**Comment by CendioOssman** [comment]
That's a shame. There are other tools though, like `man2html` that we could try.

I'm not sure what sites like die.net or man7.org use. They don't seem to have this issue.

**Comment by CendioOssman** [comment]
I don't have time to look at this, but if someone else would like to play around with it then please do.
